content/chapter-03/Summary.md

```diff
@@ -1 +1,7 @@
-# Chapter 03 - Summary
\ No newline at end of file
+# Chapter 03 - Summary
+
+* There are multiple metrics we can obtain from the system we are using
+* System metrics on the on the public cloud are more relevant than on-premise as we need to configure them directly
```

Comment:
System metrics on the public cloud are more important...

Reply:
Done